tools/format.js

```diff
@@ -0,0 +1,54 @@
+// Copyright 2018-2020 the Deno authors. All rights reserved. MIT license.
+import {
+  getPrebuiltToolPath,
+  gitLsFiles,
+  gitStaged,
+  join,
+  ROOT_PATH,
+} from "./util.js";
+
+async function getSources(baseDir, patterns) {
+  const stagedOnly = Deno.args.includes("--staged");
+
+  if (stagedOnly) {
+    return await gitStaged(baseDir, patterns);
+  } else {
+    return await gitLsFiles(baseDir, patterns);
+  }
+}
+
+async function dprint() {
+  const execPath = getPrebuiltToolPath("dprint");
+  console.log("dprint");
+  const p = Deno.run({
+    cmd: [execPath, "fmt"],
+  });
```

Comment:
This is only command that doesn't pass files explicitly, probably an overlook and it should be changed